data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/swarm/swarm_attacks_max_fraction.cfg

```diff
@@ -10,7 +10,7 @@
 # Have the side 1 leader attack the side 2 leader with its second weapon twice
 ##
 # Expected end state:
-# For both weapons, the first attack has 28 strikes and the second attack has 0 strikes (28/29, rounded down)
+# For both weapons, the first attack has 28 strikes and the second attack has 0 strikes (28/29, rounded up)
```

Comment:
> second attack has 0 strikes (28/29, rounded up

Then there would be 1 strike and test should fail. Something needs further investigation.

Reply:
i don't understand the test,i prefer don't rewrite comment

Reply:
28/29 doesn't round up to 0.

Reply:
I guess it was supposed to say 1/29 in old code too.

Reply:
replaced by "the first attack has 28 strikes (28/29, rounded up) and the second attack has 0 strikes"

Reply:
That is just wrong. First attack is 29/29.

For both weapons, the first attack has 28 strikes and the second attack has 0 strikes (1/29, rounded down)

Reply:
I suspect swarm cutoff point is still truncate, so maybe rounding comment is not correct.

Reply:
For both weapons, the first attack has 28 strikes and the second attack has 0 strikes (1/29 hp * 28 strikes gives 28/29, less than 1)

Reply:
rounded up or down when frationned value, if 27.9 then value rounded up to 28

Reply:
There is no question about rounding of 27.9. The issue is about rounding of 0.96551...

Reply:
suggestion applied

Reply:
swarm_attacks_max=27.9 not 0.965551